kfbot-refactor-phase2/src/main/java/com/twitchbotx/bot/handlers/MarathonHandler.java

```diff
@@ -159,18 +161,18 @@ public void addSub(int subPoints, boolean massGift, boolean gifted, int giftAmou
         }
 
         int pointsToAdd = 0;
-        int minutesToAdd = 0;
+        double minutesToAdd = 0;
         // adjust for  sub gifts/mass gifts
         if (massGift) {
             minutesToAdd = subPoints * subPointValue * giftAmount;
-        } else {
+       } else {
             minutesToAdd = subPoints * subPointValue;
         }
         // 1 sub point * minute value
-        pointsToAdd = minutesToAdd * 60; //convert minutes to seconds, 1 point = 1 sec
+        pointsToAdd = (int) minutesToAdd * 60; //convert minutes to seconds, 1 point = 1 sec
```

Comment:
what does this do?

Reply:
minutesToAdd variable is allowed to be a decimal, therefore it's a double, to use the addPoints method for simplicity, casting the value to an int, no values should make it here with any remainders

Reply:
cool

Reply:
to clarify this point, this will be updated to be used for specific timing in the future, namely seconds within the marathon total time, however when it reaches the addPoints() method, int is required to interact with the database

Reply:
sounds good